api/v1alpha1/nodepool_types.go

```diff
@@ -182,6 +182,9 @@ type AWSNodePoolPlatform struct {
 	// SecurityGroups is the set of security groups to associate with nodepool machines
 	// +optional
 	SecurityGroups []AWSResourceReference `json:"securityGroups,omitempty"`
+	// EBSSize specifies the volume size of the AWS EBS volume in GiBs.
+	// +optional
+	EBSSize *int32 `json:"ebsSize"`
```

Comment:
Would it be reasonable to follow the structure of `installConfig` [[reflink](https://github.com/openshift/installer/blob/24fcf7a2e962eb7d3fcca851c6c08750b3e22547/pkg/types/aws/machinepool.go#L23-L26)] for the specification of the `rootVolume`?

Reply:
Makes sense! Was wondering whether to expose type, etc along with it and this struct will encompass those.

Reply:
Actually, can we use [Volume](thirdparty/clusterapiprovideraws/v1alpha4/types.go) directly here?

Reply:
@taragu We want to keep CAPI as an implementation detail and not expose any part of its API in our external API. It'd be preferable to create a separate Volume struct even if we just copy it directly to the CAPI struct in the impl.
/cc @enxebre